_posts/2020-08-31-foreman-community-newsletter-august-2020.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: Foreman Community Newsletter (August 2020)
+date: 2020-08-31 10:49:45
+author: Foreman
+tags:
+- foreman
+---
+
+New Infrastructure SIG group, provisioning deep-dive mini survey, roadmap updates, community demos, and much more.
+
+<!--more-->
+
+Here's what happened in the community in August.
+
+## Releases
+
+### Foreman 2.1.2 has been released!
+
+This release contains six bug fixes and improvements, most notably a fix for a regression in VMWare compute resource creation via the API and dynflow support in rake tasks. For more information, see [@tbrisker's release announcement](https://community.theforeman.org/t/foreman-2-1-2-has-been-released/20230?u=mcorr).
+
+### Foreman 2.0.2 has been released!
+
+This release contains three bug fixes and improvements, most notably fixing [CVE-2020-14334](https://theforeman.org/security.html#2020-14334) and a dependency issue with `xmlrpc`.
+
+With the 2.2 release process in motion, this will most likely be the final release in the 2.0 line. Users are encouraged to start planning their upgrade process to 2.1 and beyond.
+
+For more information, see [@tbrisker's release announcement](https://community.theforeman.org/t/foreman-2-0-2-has-been-released/20161?u=mcorr).
+
+### Foreman 2.2.0-rc1 is ready for testing
+
+For the last two weeks, 2.2.0's first release candidate has been available for testing. Please install and evaluate the release candidate to help make this release as stable and successful as possible.
+
+For more information, see [@tbrisker's release candidate announcement](https://community.theforeman.org/t/foreman-2-2-0-rc1-is-ready-for-testing/20057?u=mcorr).
+
+For more information about what is included in this release, see the [release notes](https://theforeman.org/manuals/2.2/index.html#Releasenotesfor2.2).
```

Comment:
Would you mind mentioning https://community.theforeman.org/t/foreman-2-2-test-week/20122 so users can join in?